app/react/kubernetes/applications/CreateView/application-services/types.ts

```diff
@@ -8,12 +10,12 @@ export interface ServicePort {
   ingress?: object;
 }
 
-export type ServiceTypeValue = 1 | 2 | 3;
+export type ServiceTypeAngularEnum = 1 | 2 | 3; // ClusterIP | NodePort | LoadBalancer
```

Comment:
can we have constants for these service types.

Reply:
can we use `KubernetesApplicationPublishingTypes.CLUSTER_IP`

Reply:
Sure - Using KubernetesApplicationPublishingTypes.CLUSTER_IP directly gives a namespace error, but I was able to use typeof and keyof to get the values